tests/pytests/test_numbers.py

```diff
@@ -389,3 +389,106 @@ def testNegativeValues(env):
     # Query the index. if the split value of the root is nan, the query won't return any results.
     res = env.cmd('FT.SEARCH', 'idx', '@num:[-inf +inf]', 'NOCONTENT')
     env.assertEqual(res[0], doc_id)
+
```

Comment:
Test also `FT.EXPLAIN` and `FT.EXPLAINCLI` ?

Reply:
Done. [6aa6ccb](https://github.com/RediSearch/RediSearch/pull/4516/commits/6aa6ccb59067cf6c2b8491c1e4b7ce80da41516a)